client/reader/sidebar/test/promo.jsx

```diff
@@ -17,16 +56,17 @@ describe( 'ReaderSidebarPromo', () => {
 
 	test( 'should render the AppPromo when the shouldRenderAppPromo property is true', () => {
 		const adjustedProperties = { shouldRenderAppPromo: true };
-		const wrapper = shallow( <ReaderSidebarPromo { ...adjustedProperties } /> );
-		expect( wrapper.find( '.sidebar__app-promo' ) ).toHaveLength( 1 );
+		const { wrapper } = renderWithRedux( <ReaderSidebarPromo { ...adjustedProperties } /> );
+		expect( wrapper.getElementsByClassName( 'sidebar__app-promo' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should not render the AppPromo when the shouldRenderAppPromo property is false', () => {
 		const adjustedProperties = {
 			shouldRenderAppPromo: false,
 		};
-		const wrapper = shallow( <ReaderSidebarPromo { ...adjustedProperties } /> );
-		expect( wrapper.find( '.sidebar__app-promo' ) ).toHaveLength( 0 );
+		const { wrapper } = renderWithRedux( <ReaderSidebarPromo { ...adjustedProperties } /> );
+
+		expect( wrapper.getElementsByClassName( '.sidebar__app-promo' ) ).toHaveLength( 0 );
```

Comment:
This could be a good case for `toBeEmptyDOMElement()`:

```suggestion
		expect( container ).toBeEmptyDOMElement();
```

Reply:
Never used this, great one, also found some examples of this in the repo! thanks